pkg/api/types.go

```diff
@@ -1368,6 +1372,8 @@ func (p ClusterProfile) LeaseType() string {
 		return "nutanix-quota-slice"
 	case ClusterProfileOpenStack:
 		return "openstack-quota-slice"
+	case ClusterProfileOpenStackHwoffload:
+		return "openstack-hwoffload-quota-slice"
```

Comment:
You will have to create that slice in order to use this cluster profile in a job.

Reply:
done here https://github.com/openshift/release/pull/2750